src/use/use.py

```diff
@@ -832,8 +854,11 @@ def _use_str(
             importer = zipimport.zipimporter(path)
             try:
                 mod = importer.load_module(module_name)
-            except:
-                exc = traceback.format()
+            except BaseException as e:
+                if hasattr(traceback, "format"):
+                    exc = traceback.format()
```

Comment:
Hu? exc = traceback.format() is a trope, it's all over the code. Are you sure that's the only solution?